graphql-schemas/schema.yml

```diff
@@ -1122,19 +1122,23 @@
   fields:
   - { name: name, type: string, isRequired: true }
   - { name: default, type: boolean, isRequired: true }
-  - { name: weights, type: NamespaceTerraformResourceALBTargetWeights_v1, isRequired: true }
   - { name: ips, type: string, isList: true }
   - { name: openshift_service, type: string }
 
-- name: NamespaceTerraformResourceALBTargetWeights_v1
+- name: NamespaceTerraformResourceALBRules_v1
   fields:
-  - { name: read, type: int, isRequired: true }
-  - { name: write, type: int, isRequired: true }
+  - { name: condition, type: NamespaceTerraformResourceALBConditon_v1, isRequired: true}
+  - { name: action, type: NamespaceTerraformResourceALBAction_v1, isRequired: true, isList: true}
```

Comment:
AWS seem to support multiple conditions and a single action per rule. I think perhaps you meant to set `isList` to the condition element here?

Reply:
Thinking about this some more I think the current implementation is fine for the purpose of specifying both a path and methods once (vs terraform or aws implementing this as multiple conditions, that can be many things) and a list of actions in our case as we only support for forward action kind which itself supports multiple targets & weights. This is effectively abstracting and hardcoding the forward actions which is fine here

All that to say that in the end I think we don't need to change this until we may want to support other kinds of complex configurations in the future